src/app/services/socket/socket.octoprint.service.ts

```diff
@@ -227,6 +233,11 @@ export class OctoPrintSocketService implements SocketService {
         set: Math.round(message?.current?.temps[0]?.bed?.target),
         unit: '°C',
       };
+      this.printerStatus.chamber = {
```

Comment:
I'm fine with this getting slipped into this PR, but it should be mentioned in the PR text.